packages/framework/src/Twig/IconExtension.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Twig;
+
+use Twig\Environment;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class IconExtension extends AbstractExtension
+{
+    /**
+     * @var \Twig\Environment
+     */
+    protected $twigEnvironment;
+
+    /**
+     * @param \Twig\Environment $twigEnvironment
+     */
+    public function __construct(
+        Environment $twigEnvironment
+    ) {
+        $this->twigEnvironment = $twigEnvironment;
+    }
+
+    /**
+     * @return \Twig\TwigFunction[]
+     */
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction('icon', [$this, 'renderIcon'], ['is_safe' => ['html']]),
+        ];
+    }
+
+    /**
+     * @param string $iconName
+     * @param array $attributes
+     * @param string|null $iconType
+     * @return string
+     */
+    public function renderIcon(string $iconName = '', array $attributes = [], ?string $iconType = 'svg-font'): string
+    {
+        return $this->twigEnvironment->render(
+            '@ShopsysFramework/Common/Inline/Icon/icon.html.twig',
+            [
+                'name' => $iconName,
+                'attr' => $attributes,
```

Comment:
Personally I do not really like attr shortcut. Besides to be clear it should be named as `attrs` - what about not using shortcuts and name it `attributes`? 🤔

Reply:
It is consistent with naming attributes in Twig/Symfony so I would keep it.

Reply:
Well, if it is so, it should behave more like attributes in twig. See https://github.com/shopsys/shopsys/pull/2274#discussion_r595820933